prow/scripts/lib/kyma.sh

```diff
@@ -107,6 +107,35 @@ function kyma::get_last_release_version {
     fi
 }
 
+# kyma::get_previous_release_version returns previous Kyma release version (i.e. one version before the latest released version)
+#
+# Arguments:
+#   t - GitHub token
+# Returns:
+#   Previous Kyma release version
+function kyma::get_previous_release_version {
+
+    local OPTIND
+    local githubToken
+
+    while getopts ":t:" opt; do
+        case $opt in
+            t)
+                githubToken="$OPTARG" ;;
+            \?)
+                echo "Invalid option: -$OPTARG" >&1; exit 1 ;;
+            :)
+                echo "Option -$OPTARG argument not provided" >&1 ;;
+        esac
+    done
+
+    utils::check_empty_arg "$githubToken" "Github token was not provided. Exiting..."
+
+    # shellcheck disable=SC2034
+    kyma_get_previous_release_version_return_version=$(curl --silent --fail --show-error -H "Authorization: token $githubToken" "https://api.github.com/repos/kyma-project/kyma/releases" \
+        | jq -r 'del( .[] | select( (.prerelease == true) or (.draft == true) )) | sort_by(.tag_name | split(".") | map(tonumber)) | .[-2].tag_name')
```

Comment:
As discussed offline, this can be very much simplified, but for consistency sake with similar quires in the script, we will leave it like that.